spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCAuthService.java

```diff
@@ -118,33 +118,25 @@ public void call(Boolean connected) {
                     // TODO: get the email and password from encrypted storage
                     // probably want to use the KeyStore: https://developer.android.com/training/articles/keystore.html
                     if (email != null && password != null) {
-                        try {
-                            HttpHandler.getInstance()
-                                    .post(theUrl,
-                                            String.format("{\"email\": \"%s\", \"password\":\"%s\"}",
-                                                    email, password))
-                                    .subscribe(new Action1<Response>() {
-                                        @Override
-                                        public void call(Response response) {
-                                            try {
-                                                accessToken = new JSONObject(response.body().string()).getString("token");
-                                                if (accessToken != null) {
-                                                    loginStatus.onNext(1);
-                                                }
-                                            }
-                                            catch (JSONException e) {
-                                                e.printStackTrace();
-                                            } catch (IOException ee) {
-
-                                            }
+                        HttpHandler.getInstance()
+                            .post(theUrl, String.format("{\"email\": \"%s\", \"password\":\"%s\"}", email, password))
+                            .subscribe(new Action1<Response>() {
+                                @Override
+                                public void call(Response response) {
+                                    try {
+                                        accessToken = new JSONObject(response.body().string()).getString("token");
```

Comment:
Good catch....we're going to wait on updating the native SDKs until that branch is merged into the develop branch of spatialconnect-server